cds_ils/migrator/api.py

```diff
@@ -16,28 +16,37 @@
 from elasticsearch import VERSION as ES_VERSION
```

Comment:
Not for this PR: I would re-organize the code because it looks like that it is growing... e.g. class Patron with get and import methods, etc...

Reply:
I started already :)